dashboard/src/components/providers/AppProvider.tsx

```diff
@@ -110,6 +110,31 @@ const initializingQueries = `
   from iasql_tables as t;
 `;
 
+const gettingStarted = `-- Welcome to IaSQL!
+
+-- You can write any SQL query here and run it using the "Run query" button on the top
+-- right or by using the Ctrl+Enter (CMD+Return on Mac) keyboard shortcut.
+
+-- Open as many tabs as you want and run queries on each of them.
+
+-- You can start by installing a module by clicking the "Install" button from the modules list on the sidebar.
+
+-- Going to the sidebar's "Schema" tab you can see the tables and columns of the installed modules. There you
+-- can check the documentation of each module. You can also do a quick select query on any table with data.
+
+-- This editor can autocomplete your query's table and column names based on the installed modules.
+
+-- If you want to see all the available IaSQL functions you can execute the following query:
+-- SELECT * FROM iasql_help()
+
+-- To disconnect your database, click "Settings" and then "Disconnect" on the top right.
```

Comment:
Oh, I thought we were going to unwind the settings dropdown? Did we decide against that?

Reply:
I don't think we've decided against it. I was doing the task of removing export buttons and maybe I should've done it in my PR. I can submit another PR removing it.